fdbclient/include/fdbclient/BlobCipher.h

```diff
@@ -202,6 +205,15 @@ struct hash<BlobCipherDetails> {
 };
 } // namespace std
 
+struct EncryptHeaderCipherDetails {
```

Comment:
am confused: this should have been already present on `main`?

Reply:
hmm, must be merge conflict.

Reply:
It is just being move from line 334 to here.